src/ResourceManager/Sql/Commands.Sql/ChangeLog.md

```diff
@@ -28,6 +28,12 @@
 * Clarified User-Defined Restore Points for SQLDW in New-AzureRmSqlDatabaseRestorePoint help
 * Fixed formatting of OutputType in help files
 * Updated documentation of -ComputeGeneration parameter in several cmdlets
+* Adding Server Advanced Threat Protection support with the following cmdlets:
```

Comment:
@yaakoviyun These changelog entries need to be moved up under the `Current Release` section.